import java.util.Scanner;

class FeetDistanceConverter {
    void convert() {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter distance in feet: ");
        double feet = sc.nextDouble();

        double yards = feet / 3;
        double miles = yards / 1760;

        System.out.println("Distance in yards is " + yards +
                " and miles is " + miles);
    }

    public static void main(String[] args) {
        FeetDistanceConverter obj = new FeetDistanceConverter();
        obj.convert();
    }
}